src/test/java/io/getstream/client/FeedTest.java

```diff
@@ -140,6 +144,28 @@ public void unfollow() throws Exception {
     feed1.unfollow(feed2).join();
   }
 
+  @Test
+  public void getFollowStats() throws Exception {
+    Client client =
+        Client.builder(apiKey, secret)
+            .httpClient(new OKHTTPClientAdapter(new OkHttpClient()))
+            .build();
+    String feed1Id = "flat:1";
+    FlatFeed feed1 = client.flatFeed("flat", "1");
+    FlatFeed feed2 = client.flatFeed("flat", "2");
+    feed1.follow(feed2).join();
+
+    FollowStats stats =
+        feed1.getFollowStats(Collections.emptyList(), Lists.newArrayList("timeline")).join();
+    assertEquals(0, stats.getFollowers().getCount());
+    assertEquals(feed1Id, stats.getFollowers().getFeed());
+
+    stats = feed1.getFollowStats(Collections.emptyList(), Lists.newArrayList("user")).join();
+    assertEquals(0, stats.getFollowers().getCount());
+    assertEquals(feed1Id, stats.getFollowers().getFeed());
+    assertEquals(feed1Id, stats.getFollowing().getFeed());
+  }
+
```

Comment:
can we setup the data of this test here with unique feeds instead of relying on existing data ?

also, checking counts would be needed

Reply:
yeah I can. I was a bit confused why the rest of the tests are using existing data

Reply:
legacy

Reply:
we need to update them at same point too